### ISSUE TITLE
Eclipse splash image looks bad on macOS

### ISSUE THREAD

**Issue by kgibm** [eclipse-platform/eclipse.platform#39]
I'm a committer on the [Eclipse Memory Analyzer Tool project](https://www.eclipse.org/mat/) and we recently added a splash screen to our [standalone builds](https://www.eclipse.org/mat/downloads.php). This works fine on Windows and Linux, but on macOS x64, it's very strange: first, it shows for a second with the right dimensions but very low quality and then it switches to an image with the right quality but it is patterned multiple times. Here is a GIF of the start-up sequence (it takes a few seconds to start):

![matsplash](https://user-images.githubusercontent.com/6618433/175061074-7315154b-af54-498c-8b19-bd7f40ae7f36.gif)

The `splash.bmp` is under `plugins/org.eclipse.mat.ui.rcp/splash.bmp`:

1. `git clone https://git.eclipse.org/r/mat/org.eclipse.mat.git`
2. `ls org.eclipse.mat/plugins/org.eclipse.mat.ui.rcp/splash.bmp`

Here is the original and it looks this good on Windows and Linux (converted to `jpeg` as GitHub doesn't support `bmp` uploads):

![splash](https://user-images.githubusercontent.com/6618433/175062323-6a24dac4-70c9-490d-938d-09b058afe765.jpeg)

The image was created using Gimp on macOS and here is the Gimp file (in a `zip` for GitHub upload):

[matsplash.zip](https://github.com/eclipse-platform/eclipse.platform/files/8958866/matsplash.zip)

As per the [Eclipse Splash Screen guidance](https://wiki.eclipse.org/Platform-releng/Updating_Branding#Splash_screen), the bmp was created without color space information and in 24-bit format.

The `product` file that specifies the splash image is here: https://git.eclipse.org/c/mat/org.eclipse.mat.git/tree/org.eclipse.mat.product/mat-2022-03.product

```
   <splash
      location="org.eclipse.mat.ui.rcp"
      startupProgressRect="5,275,445,15"
      startupMessageRect="7,252,445,20"
      startupForegroundColor="000000" />
```

That project has: https://git.eclipse.org/c/mat/org.eclipse.mat.git/tree/plugins/org.eclipse.mat.ui.rcp/config.ini

```
osgi.splashPath=platform:/base/plugins/org.eclipse.mat.ui.rcp
```

If you are running on macOS, you can reproduce the problem like so:

1. Make sure Java 11 is on your `PATH`
2. `git clone https://git.eclipse.org/r/mat/org.eclipse.mat.git`
3. `cd org.eclipse.mat/parent`
4. `find .. -type f -name "*mat*product" -not -name mat-2022-03.product -delete`
5. `mvn -DskipTests=true clean install`
6. `open -n ../org.eclipse.mat.product/target/products/org.eclipse.mat.ui.rcp.MemoryAnalyzer/macosx/cocoa/x86_64/mat.app`
7. Observe the strange splash screen

**Comment by sravanlakkimsetti** [comment]
I have seen this kind of behavior in Windows with 125% scaling factor. By the way you can use png files as splash screens now.

**Comment by sravanlakkimsetti** [comment]
The problem is the the splash screen image. The splash screen was designed with 53 dpi instead of 72 dpi which was required on Mac. I updated the splash to 72 dpi and the problem disappeared.

I used gimp for the conversion.

1. Load the image in gimp
2. use image scale and select dpi as 72.
3. export the image appropriately

here is the png image I converted 
![splash](https://user-images.githubusercontent.com/77106773/176618268-222ad069-776d-4df5-88ea-018935425a4d.png)
